mp4parse/src/tests.rs

```diff
@@ -388,6 +388,36 @@ fn read_vpcc() {
     assert!(r.is_ok());
 }
 
+// TODO: it'd be better to find a real sample here.
+#[test]
+fn read_vpcc_version_1() {
+    let data_length = 12u16;
+    let mut stream = make_fullbox(BoxSize::Auto, b"vpcC", 1, |s| {
+        s.B8(2)     // profile
+         .B8(0)     // level
+         .B8(0x82)  // bitdepth (4 bits), chroma (3 bits), video full range (1 bit)
+         .B8(0)     // color primaries
+         .B8(0)     // transfer characteristics
+         .B8(0)     // matrix
```

Comment:
ISO 23001-8 is [available](http://standards.iso.org/ittf/PubliclyAvailableStandards/index.html) without charge. It looks like `1` (BT.709) would work fine for the matrix field.